### ISSUE TITLE
Fix the range of an axis

### ISSUE THREAD

**Issue by robocop** [SiegeLord/RustGnuplot#10]
Hello, 

First at all, thank you for your work on this library!

I have a question about fixing the range of an axis.

I tried: 

    let mut fg = Figure::new();
    fg.axes2d().set_x_range(gnuplot::AutoOption::Fix(0.0), gnuplot::AutoOption::Fix(1.0));
    fg.axes2d().lines_points(&times, &s1, &[Color("red"), PointSymbol('.')]);
    fg.axes2d().lines_points(&times, &s2, &[Color("blue"), PointSymbol('.')]);

But it seems that it does not work (I got two different ranges for the x axis, which is not very beautiful...).

Any idea?

Thanks

**Comment by SiegeLord** [comment]
Every time you call `axes2d` you're actually creating a separate set of axes. Since they all start in the same spot by default, you get the overlapping behaviour. What you probably want is this:

```rust
let mut fg = Figure::new();
fg.axes2d()
  .set_x_range(gnuplot::AutoOption::Fix(0.0), gnuplot::AutoOption::Fix(1.0))
  .lines_points(&times, &s1, &[Color("red"), PointSymbol('.')]);
  .lines_points(&times, &s2, &[Color("blue"), PointSymbol('.')]);
```

**Comment by robocop** [comment]
Thank you for your answer !
